src/server/auth/middleware/bearerAuth.ts

```diff
@@ -73,7 +73,14 @@ export function requireBearerAuth({ verifier, requiredScopes = [], resourceMetad
       req.auth = authInfo;
       next();
     } catch (error) {
-      if (error instanceof InvalidTokenError) {
+      if (error instanceof MissingAuthenticationError) {
+        // RFC 6750 Section 3.1: Missing authentication should not include error codes
+        const wwwAuthValue = resourceMetadataUrl
+          ? `Bearer realm="protected", resource_metadata="${resourceMetadataUrl}"`
```

Comment:
```suggestion
          ? `Bearer resource_metadata="${resourceMetadataUrl}"`
```
Realm is a MAY and is not adding much here as its only on this one, and is not configurable by the user.. let's drop it.